lua/lazyvim/plugins/extras/util/chezmoi.lua

```diff
@@ -0,0 +1,70 @@
+return {
+  {
+    -- highlighting for chezmoi files template files
+    "alker0/chezmoi.vim",
+    init = function()
+      vim.g["chezmoi#use_tmp_buffer"] = 1
+      vim.g["chezmoi#source_dir_path"] = os.getenv("HOME") .. "/.local/share/chezmoi"
+    end,
+  },
+  {
+    "xvzc/chezmoi.nvim",
+    dependencies = { "nvim-lua/plenary.nvim" },
```

Comment:
this dep is not needed, so please remove

Reply:
This dependency is provided in the [installation docs](https://github.com/xvzc/chezmoi.nvim#installation). Or do you mean that Lazyvim somehow provides this dep to all packages?

Reply:
already included. just remove it